FEXCore/Source/Interface/Core/Frontend.cpp

```diff
@@ -315,6 +323,12 @@ void Decoder::DecodeModRM_64(X86Tables::DecodedOperand* Operand, X86Tables::ModR
 }
 
 bool Decoder::NormalOp(const FEXCore::X86Tables::X86InstInfo* Info, uint16_t Op, DecodedHeader Options) {
+  if (Info->Type == FEXCore::X86Tables::TYPE_ARCH_DISPATCHER) [[unlikely]] {
+    // Dispatcher Op.
+    // TODO: Move this in to `NormalOpHeader`, Dispatch tables have a bug currently where some subtables don't inherit flags correctly.
```

Comment:
What is the bug? Does it need to be fixed before merging? If not, how can it be reproduced?

Comments mentioning a quirk should have a clear reference or explanation that defines what quirk needs to be addressed before the comment can be removed.

Reply:
Removing the code and running our ASM tests is enough to catch the issue. It doesn't need to be fixed first, mostly because it's going to be involved walking through all of our tables and ensuring flags are inherited correctly.

Reply:
Extended the comment to explain running the asm tests is enough to reproduce the issue.